### PR TITLE
Bug 1878014, updated note on block storage volumes

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1878014

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@bobfuru Thanks for your insight and previous work here! Can you PTAL? Thanks!

**Comment by ahardin-rh** [comment]
@wzheng1 Can you please provide QE review? Thanks!

File: modules/installation-registry-storage-block-recreate-rollout.adoc [hunk 1]
```diff
@@ -16,7 +16,10 @@ To allow the image registry to use block storage types such as vSphere Virtual M
 
 [IMPORTANT]
 ====
-Block storage volumes are not recommended for use with image registry on production clusters. An installation where the registry is configured on block storage is not highly available because the registry cannot have more than one replica.
+Block storage volumes are *supported but not recommended* for use with image
```
Comment (bobfuru) [inline modules/installation-registry-storage-block-recreate-rollout.adoc:19]: I actually meant the asterisks in the BZ comment to just highlight my suggested wording change. I don't think it looks all that bad included here, though it does go against our style guidelines so I'd be inclined to remove the bolding. WDYT?

**Review by bobfuru — COMMENTED** [review]
Thanks for handling this, @ahardin-rh! Just one comment about formatting and otherwise LGTM!

**Comment by wzheng1** [comment]
Ack for the description!

File: modules/installation-registry-storage-block-recreate-rollout.adoc (hunk 1 above)
Comment (ahardin-rh) [reply modules/installation-registry-storage-block-recreate-rollout.adoc]: I flip-flopped a bit, but removed the asterisks to be more in line with our style guidelines. Thanks!

**Pull request merged**